perses/rjmc/topology_proposal.py

```diff
@@ -787,6 +791,10 @@ class TopologyProposal(object):
         Name of the old residue
     new_residue_name : str
         Name of the new residue
+    _old_networkx_residue : NetworkXMolecule
```

Comment:
we also pass networkx representations of molecules to the `TopologyProposal` so that we can easily perceive chirality, tally missing terms with ring closure, and i'm sure other useful applications

Reply:
Instead of this, what if we subclass `Topology` to store this information in a convenience method, like `topology.get_networkx_residue(resid)` or `Residue.to_networkx()`?  This would prevent us from having to change _every_ call to `TopologyProposal()` (which is a _LOT_) and would position us to easily swap out the OpenMM `Topology` object with the openforcefield `Topology` object once we add biopolymer support in a few days.